source/installation-guide/installing-wazuh-server/sources_installation.rst

```diff
@@ -26,11 +24,40 @@ Installing Wazuh manager
 
       # apt-get install python gcc make libc6-dev curl policycoreutils automake autoconf libtool
 
+    These are the basic dependencies to build Wazuh's binaries if you are going to install them in the default directory but, if you want to install in any other directory, you need to install some new dependencies in order to build the CPython interpreter included in the Wazuh manager installation since v3.9.0. By default, ``make deps`` will download a pre-compiled version of CPython, built to be installed in ``/var/ossec``, otherwise, it will download a modified version of CPython sources. This sources will build CPython for the target directory and link the interpreter and the internal dependencies against the ``libwazuhext``.
+
+    To install the build dependencies of CPython, you need to add the sources repository for Debian/Ubuntu based OS if they are not already present and then, install those dependencies:
```

Comment:
Debian / Ubuntu? Do you mean Debian/RPM?

Reply:
@jesuslinares actually, installing the sources repository is needed for Debian and Ubuntu based OS but, for RHEL/CentOS based OS, this isn't necessary. I will explain better this in the docs.